.github/workflows/continuous-integration.yml

```diff
@@ -6,14 +6,12 @@ on:
       - main
   pull_request:
 
-env:
-  DOCKER_REGISTRY: ghcr.io
-  DOCKER_IMAGE_NAME: ${{ github.repository }}
-
 jobs:
   codecov:
     name: codecov
     runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: true
```

Comment:
Needed to work around the issue mentioned in https://github.com/moby/moby/issues/37965.